api/v1/requests/EventCreationRequest.js

```diff
@@ -1,28 +1,27 @@
 var Request = require('./Request');
 var validators = require('../utils/validators');
-var events = require('../utils/events');
+var time = require('../utils/time');
 
 var eventLocationValidations = {
     locationId:  ['required', 'integer']
 };
 
-var bodyRequired = ['event', 'eventLocations'];
+var bodyRequired = ['event'];
+var bodyAllowed = ['eventLocations']
 var bodyValidations = {
     'event': ['required', 'plainObject'],
     'event.name': ['required', 'string', 'maxLength:255'],
-    'event.shortName': ['required', 'string', 'maxLength:25'],
     'event.description': ['required', 'string', 'maxLength:2047'],
-    'event.tracking': ['required', 'boolean'],
-    'event.startTime': ['required', events.verifyDate],
-    'event.endTime': ['required', events.verifyDate],
-    'event.tag': ['required', 'string', events.verifyTag],
```

Comment:
Was this in reference to just the tag parameter, or the tracking boolean as well?